cmd/bigsky/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Run this dockerfile from the top level of the indigo git repository like:
+#
+#   podman build -f ./cmd/bigsky/Dockerfile -t bigsky .
+
+### Compile stage
+
+# trouble with 'No such file or directory' when building using 'golang:1.19'
+# images, I think because of library errors when copied to 'alpine' image below
+# https://megamorf.gitlab.io/2019/09/08/alpine-go-builds-with-cgo-enabled/
+# TODO: lock to a specific version of go toolchain
+FROM alpine:3.17 AS build-env
+RUN apk add --no-cache build-base make go
+
+ADD . /dockerbuild
+WORKDIR /dockerbuild
+
+RUN go build -o /bigsky ./cmd/bigsky
+
+### Run stage
+FROM alpine:3.17
+
+RUN apk add --no-cache --update dumb-init
+ENTRYPOINT ["dumb-init", "--"]
+
+WORKDIR /
+RUN mkdir -p data/bigsky
```

Comment:
We'll mount a volume like `--volume=/data:/data` into the container which will mask any directories created  in here. We should also be able to specify a `BIGSKY_DATA_DIR=/data/bigsky` or similar env I think, so that it's not hard coded.

Reply:
Yup, have an env var to move this. And the current implementation will create the directory structure if needed on boot (`mkdir -p` equivalent). Partially just put that in as documentation.

`bigsky` mostly uses this for "carstore" (repo storage on disk as CAR files), but by default it will also use sqlite database at this location. It will only look for postgresql if configured to do so via database connection string.